frontend/src/components/details/CanvasComponent.jsx

```diff
@@ -27,41 +26,41 @@ class CanvasComponent extends Component {
         a canvas page without any comparisons 
         */
             return (
-                <Container fluid={true}>
-                    <Row>
-                        <Col xs={5} style={{ marginTop: 30 }}>
```

Comment:
xs={5} and md={10} can also be used to react to different screen sizes @gandompm @IremToroslu @manifau 🤔 so why swap to w3 classes in this case?